### ISSUE TITLE
Support filter on filename in the file tree example

### ISSUE THREAD

**Issue by echarles** [jpmorganchase/regular-table#43]
While showcasing file tree example, can we demonstrate how to add a filter on file names like discussed on https://github.com/jupyterlab/jupyterlab/issues/8157 ?

**Comment by telamonian** [comment]
Absolutely. My plan for this is to add a breadcrumbs/filter UI component to the top of the filename column. Think along the lines of github's own "find file" UI:

![github_file_search_ui](https://user-images.githubusercontent.com/2263641/84058708-96913800-a987-11ea-9603-752ae249ddda.gif)

**Comment by sakshamk6999** [comment]
Hi, I am interested in contributing for this issue. So, we need to add a filter functionality right ?
And would this feature be limited to the file tree example only ?

**Comment by sakshamk6999** [comment]
There is an error, it shows nothing in the html files in the example directory

**Comment by awasthishubh** [comment]
Hey @sakshamk6999, are you still working on this?

**Comment by harshilsharma1997** [comment]
Hey, I have been looking at this issue for sometime, and I was wondering, if we can just use the regular-table element like the normal table element, and add a filter component over it, just like we can in a table element ?
